src/components/Footer/Footer.jsx

```diff
@@ -39,7 +42,7 @@ function Footer() {
         </div>
 
         <Typography elType="body1" className={styles.copyright}>
-          Copyright 2020. All Rights Reserved
+          Copyright {thisYear}. All Rights Reserved
```

Comment:
As per the design, the value should not be updated. Is it mentioned in any requirement that this year will be dynamic?

Reply:
I assumed it needed to be done since there was a ticket for it: https://poulamisaha.atlassian.net/jira/software/projects/KAN/boards/1?selectedIssue=KAN-20

@poulami-saha Please confirm that I should close the PR.

Reply:
Discussion on Slack seems to agree that we will keep the static year from the design. I would close the PR. As long as you don't delete your branch, it's very easy to open a new one for it if we later change our minds.